.github/workflows/elixir.yml

```diff
@@ -68,10 +68,17 @@ jobs:
         otp: ['22', '23', '24', '25', '26']
         elixir: ['1.11', '1.12', '1.13', '1.14', '1.15', '1.16']
         exclude:
-          - {otp: '25', elixir: '1.10'}
+          - {otp: '22', elixir: '1.14'}
+          - {otp: '22', elixir: '1.15'}
+          - {otp: '22', elixir: '1.16'}
+          - {otp: '23', elixir: '1.14'}
+          - {otp: '23', elixir: '1.15'}
+          - {otp: '23', elixir: '1.16'}
+          - {otp: '24', elixir: '1.14'}
+          - {otp: '24', elixir: '1.15'}
+          - {otp: '24', elixir: '1.16'}
           - {otp: '25', elixir: '1.11'}
           - {otp: '25', elixir: '1.12'}
-          - {otp: '26', elixir: '1.10'}
           - {otp: '26', elixir: '1.11'}
```

Comment:
1.11 is not supported anymore if I read this correctly:

https://hexdocs.pm/elixir/1.16.2/compatibility-and-deprecations.html

Maybe we can remove an extra one?